react/features/invite/components/add-people-dialog/web/InviteByEmailSection.tsx

```diff
@@ -157,26 +157,18 @@ function InviteByEmailSection({ inviteSubject, inviteText, inviteTextiOS, t }: P
 
     return (
         <>
-            <div>
-                <div
-                    aria-expanded = { isActive }
-                    aria-label = { t('addPeople.shareInvite') }
-                    className = { `invite-more-dialog email-container${isActive ? ' active' : ''}` }
-                    onClick = { _onToggleActiveState }
-                    onKeyPress = { _onToggleActiveStateKeyPress }
-                    role = 'button'
-                    tabIndex = { 0 }>
-                    <span>{t('addPeople.shareInvite')}</span>
-                    <Icon src = { IconArrowDownSmall } />
-                </div>
-                <div className = { `invite-more-dialog icon-container${isActive ? ' active' : ''}` }>
+            <div className = { classes.container }>
+                <p className = { classes.label }>{t('addPeople.shareInvite')}</p>
+                <div className = { classes.iconRow }>
                     <Tooltip
                         content = { t('addPeople.copyInvite') }
                         position = 'top'>
                         <div
                             aria-label = { t('addPeople.copyInvite') }
-                            className = 'copy-invite-icon'
+                            className = { classes.iconContainer }
+                            // eslint-disable-next-line react/jsx-no-bind
```

Comment:
doesn't it make sense to avoid these with useCallback ?

Reply:
Yes and no. I think we should actually revisit this rule. In some cases (event handlers like this), where we would have an empty dep array, useCallback is pure overhead. It may actually hurt performance (the hook is called on each re-render)

Reply:
What if that callback is on an item on a list of 500 such items?

Reply:
I don't think there's any performance benefit. Instead of calling the hook and comparing the dep array (which is empty) we just re-declare a function that just calls another function

Reply:
So, IIRC, and correct me if I'm wrong, the case we originally wanted to avoid is the following:

- Have a container component with some "handle item" like function
- Have that container create N child component, for a large N
- The container passes that function to each child as a prop
- Binding the function up front would mean we have 1 instance of the function in use, reused by all N children
- The opposite approach would create N function bindings, which can contribute to putting pressure on the GC